mongo_connector/__init__.py

```diff
@@ -11,7 +11,23 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+import logging
+
 from pkgutil import extend_path
 __path__ = extend_path(__path__, __name__)
 
-__version__ = '2.5.0.dev0'
+version_info = (2, 5, 0, 'dev0')
+version = '.'.join(str(v) for v in version_info)
+__version__ = version
```

Comment:
Why both `version` and `__version__`? I think I've seen `__version__` more often in projects.